service/history/historyEngine.go

```diff
@@ -2110,6 +2110,9 @@ func (e *historyEngineImpl) SignalWorkflowExecution(
 	if err != nil {
 		return err
 	}
+	if domainEntry.GetInfo().Status != persistence.DomainStatusRegistered {
```

Comment:
I thought we discussed something about creating a dynamic config for this behavior? I am not sure what's the final decision we made for that?

Reply:
I don't think it's worth the effort.